### DISCUSSION TITLE
Problem filtering a url field

### DISCUSSION THREAD

**Discussion by kabapy** [pocketbase/pocketbase#4265] [Q&A]
Hello,
I am having a problem filtering a url field.
This works
`source_url="https://www.youtube.com/watch?v=IbyVCCC64D4"`

This do not work, yielding no records
`
source_url="http://www.elsport.com/news/show/721608/%D9%85%D9%88%D9%82%D8%B9%D8%A9-%D9%86%D8%A7%D8%B1%D9%8A%D8%A9-%D8%AF%D9%88%D8%B1-%D8%A7%D9%8416-%D8%A8%D9%8A%D9%86-%D9"
`
![Screenshot_20240130-115313_Chrome](https://github.com/pocketbase/pocketbase/assets/28626035/25af6999-8c8d-470d-9a49-7af28368cd80)

Any thoughts

**Comment by ganigeorgiev** [comment]
`source_url = "some string"` performs an exact string match, meaning that the stored field value must be the same as the one you are filtering (_note that there is no url decoding if that is what you are expecting_).

If your request returns an error, you can try to start the executable with `--dev` for more detailed console messages and/or check the request logs in the _Admin UI > Logs_ list.

**Reply by kabapy** [reply]
I copied the string In the admin ui, and I am trying to filter the field in the admin ui also

**Reply by ganigeorgiev** [reply]
@kabapy I've just tested it on the live demo and it works fine for me:
https://pocketbase.io/_/#/collections?collectionId=irqyv9ewy7sw3ra&filter=source_url+%3D+%22http%3A%2F%2Fwww.elsport.com%2Fnews%2Fshow%2F721608%2F%25D9%2585%25D9%2588%25D9%2582%25D8%25B9%25D8%25A9-%25D9%2586%25D8%25A7%25D8%25B1%25D9%258A%25D8%25A9-%25D8%25AF%25D9%2588%25D8%25B1-%25D8%25A7%25D9%258416-%25D8%25A8%25D9%258A%25D9%2586-%25D9%22&sort=-created

_And also a direct API link: https://pocketbase.io/api/collections/irqyv9ewy7sw3ra/records?filter=source_url%20%3D%20%22http%3A%2F%2Fwww.elsport.com%2Fnews%2Fshow%2F721608%2F%25D9%2585%25D9%2588%25D9%2582%25D8%25B9%25D8%25A9-%25D9%2586%25D8%25A7%25D8%25B1%25D9%258A%25D8%25A9-%25D8%25AF%25D9%2588%25D8%25B1-%25D8%25A7%25D9%258416-%25D8%25A8%25D9%258A%25D9%2586-%25D9%22_

(_note that the demo resets every hour or so_)

**Comment by kabapy** [comment]
On further testing, I think there are some hidden characters at the end of the url. It worked with a wildcard at the end !
